compiler/rustc_codegen_ssa/src/target_features.rs

```diff
@@ -43,10 +43,8 @@ const ARM_ALLOWED_FEATURES: &[(&str, Option<Symbol>)] = &[
 ];
 
 const AARCH64_ALLOWED_FEATURES: &[(&str, Option<Symbol>)] = &[
-    // FEAT_AdvSimd
+    // FEAT_AdvSimd & FEAT_FP
```

Comment:
Can't comment, but should remove the entry in `AARCH64_TIED_FEATURES` on line 144.

Reply:
Weird, I thought I did.
I am going to chalk that up to `git` working overtime to earn its name.